rllib/evaluation/postprocessing.py

```diff
@@ -207,26 +222,70 @@ def compute_gae_for_sample_batch(
             #  running across different processes for different trajectories?
             #  This implementation right now will compute even the action_dist which
             #  will not be needed but takes time to compute.
+            # In order to calculate the batch size ad hoc, we need a sample batch.
+            if not isinstance(input_dict, SampleBatch):
+                input_dict = SampleBatch(input_dict)
+            seq_lens = np.array([1] * len(input_dict))
+            input_dict = policy.maybe_add_time_dimension(input_dict, seq_lens=seq_lens)
```

Comment:
The changes in this file revolve around the problem that, for a recurrent RLModule, the output for forward exploration will have a time dimension.
So vf_preds will also have a time dimension.